pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -169,66 +176,178 @@ pub mod pallet {
 	impl<T: Config> Pallet<T> {
 		#[pallet::call_index(0)]
 		#[pallet::weight(Weight::from_parts(10_000, 0) + T::DbWeight::get().writes(1))]
-		pub fn create_pool(_origin: OriginFor<T>) -> DispatchResult {
-			todo!()
+		pub fn create_pool(
+			origin: OriginFor<T>,
+			curve: CurveInput<CurveParameterInputOf<T>>,
+			currencies: BoundedVec<TokenMetaOf<T>, T::MaxCurrencies>,
+			denomination: u8,
+			pool_manager: AccountIdOf<T>,
```

Comment:
I think we wanted to move this to its own extrinsic? (`set_manager` maybe?)

Reply:
Ah okay. I would maybe wait for this change, until the [wireframe PR](https://github.com/KILTprotocol/kilt-node/pull/772) is merged. Otherwise, I get some unnecessary merge conflicts.

Reply:
Added tx in: [7f9a319](https://github.com/KILTprotocol/kilt-node/pull/768/commits/7f9a31966ead7d4e74c6acf20ff5e0cf968e3a36)

Reply:
Or should I check, if who is owner? Should the owner be able to update the manager, or should the manager have the power?

Reply:
I would argue, the manager should have the power. Follows the logical flow of a root in substrate

Reply:
The manager should be able to set the new manager. This follows from our idea to have manager-less pools, where the owner is set to None - and then can never be re-set again. This way users can be certain that there is no account that has special powers for the pool.